stylesheets/main.css

```diff
@@ -60,7 +68,7 @@ div.menu nav ul {
     box-sizing: border-box;
     height: 100%;
     left: 0px;
-    position: absolute;
+    position: fixed;
```

Comment:
Why fixed? Won't this interfere with any other `.full-vp-bg-image` `div`s we make?

Reply:
Ah - I saw that this class was being used for the header image so I applied the change here.

If we intend to use `.full-vp-bg-image` for other images, then I agree that the position shouldn't be applied here. I can create a new class, called `.landing-header-image`, and apply the `position: fixed` there.

The `position: fixed` is to achieve the effect of the text gliding above the image as you scroll.

What do you think?

Reply:
Oh right, I didn't like that...
I give @anitasunshine the final say on this.

Reply:
Sorry Jeffrey, after looking at it some more, it is a little funny looking when it moves and our eyes are peering over the line. If it is another picture maybe we can utilize this

Reply:
No worries! I'll change it back to absolute.